### ISSUE TITLE
"Watching Disabled" Status

### ISSUE THREAD

**Issue by sachinrane** [kartikey54/VaccineAvailabilityNotifier#25]
![image](https://user-images.githubusercontent.com/34787376/116825292-41b46c80-abac-11eb-9e13-f293b9f94b64.png)

It's running fine but I get the status of "disabled" for the 'watching' status. 

Does anything need to be done to resolve this?
@kartikey54

**Comment by ghaisasadvait** [comment]
same issue for me
![image](https://user-images.githubusercontent.com/11556546/116831107-112ffb00-abcb-11eb-8415-a280279b1f31.png)

**Comment by jarvishu** [comment]
Yup same for me. Please drop the comment if someone find out the solution 
Not getting sample mail too.

**Comment by jarvishu** [comment]
> Yup same for me. Please drop the comment if someone find out the solution
> Not getting sample mail too.

### Fixed the issue
1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
2> run command **pm2 start vaccineNotifier.js --watch**

**Comment by sid8840** [comment]
> > Yup same for me. Please drop the comment if someone find out the solution
> > Not getting sample mail too.
> 
> ### Fixed the issue
> 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> 2> run command **pm2 start vaccineNotifier.js --watch**

even after watching is Enabled, did you get any email??

**Comment by Soniya318** [comment]
> > Yup same for me. Please drop the comment if someone find out the solution
> > Not getting sample mail too.
> 
> ### Fixed the issue
> 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> 2> run command **pm2 start vaccineNotifier.js --watch**

![image](https://user-images.githubusercontent.com/74868644/116844170-f1ff9080-abff-11eb-8bae-b9f389e0685f.png)

not getting emails

**Comment by sid8840** [comment]
> > > Yup same for me. Please drop the comment if someone find out the solution
> > > Not getting sample mail too.
> > 
> > 
> > ### Fixed the issue
> > 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> > 2> run command **pm2 start vaccineNotifier.js --watch**
> 
> ![image](https://user-images.githubusercontent.com/74868644/116844170-f1ff9080-abff-11eb-8bae-b9f389e0685f.png)
> 
> not getting emails

instead type this
npm i && pm2 start vaccineNotifier.js --watch

**Comment by Soniya318** [comment]
> > > > Yup same for me. Please drop the comment if someone find out the solution
> > > > Not getting sample mail too.
> > > 
> > > 
> > > ### Fixed the issue
> > > 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> > > 2> run command **pm2 start vaccineNotifier.js --watch**
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/74868644/116844170-f1ff9080-abff-11eb-8bae-b9f389e0685f.png)
> > not getting emails
> 
> instead type this
> npm i && pm2 start vaccineNotifier.js --watch

do we need to replace 'slotDetails' with 'validSlots' in notifier.sendEmail call :?

**Comment by sid8840** [comment]
> > > > > Yup same for me. Please drop the comment if someone find out the solution
> > > > > Not getting sample mail too.
> > > > 
> > > > 
> > > > ### Fixed the issue
> > > > 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> > > > 2> run command **pm2 start vaccineNotifier.js --watch**
> > > 
> > > 
> > > ![image](https://user-images.githubusercontent.com/74868644/116844170-f1ff9080-abff-11eb-8bae-b9f389e0685f.png)
> > > not getting emails
> > 
> > 
> > instead type this
> > npm i && pm2 start vaccineNotifier.js --watch
> 
> do we need to replace 'slotDetails' with 'validSlots' in notifier.sendEmail call :?

yes

or us this PR
https://github.com/jithin-zachariah/VaccineAvailabilityNotifier

**Comment by Soniya318** [comment]
> > > > > > Yup same for me. Please drop the comment if someone find out the solution
> > > > > > Not getting sample mail too.
> > > > > 
> > > > > 
> > > > > ### Fixed the issue
> > > > > 1> Stop the process (**pm2 stop vaccineNotifier.js && pm2 delete vaccineNotifier.js**)
> > > > > 2> run command **pm2 start vaccineNotifier.js --watch**
> > > > 
> > > > 
> > > > ![image](https://user-images.githubusercontent.com/74868644/116844170-f1ff9080-abff-11eb-8bae-b9f389e0685f.png)
> > > > not getting emails
> > > 
> > > 
> > > instead type this
> > > npm i && pm2 start vaccineNotifier.js --watch
> > 
> > 
> > do we need to replace 'slotDetails' with 'validSlots' in notifier.sendEmail call :?
> 
> yes
> 
> or us this PR
> https://github.com/jithin-zachariah/VaccineAvailabilityNotifier

it finally worked! thank you :)

**Comment by Pratik2125** [comment]
1) made the changes in notifier (i.e replaced 'slotDetails' with 'validSlots')
2) Entering the age for which cowin website is showing dates near the pincode.
3) Still not getting mails.

<img width="1420" alt="Slots" src="https://user-images.githubusercontent.com/16135238/116850440-4c9fe900-ac0e-11eb-922e-86cc884098bd.png">

Any help

**Comment by sid8840** [comment]
Are slots available on Cowin site??

> 1. made the changes in notifier (i.e replaced 'slotDetails' with 'validSlots')
> 2. Entering the age for which cowin website is showing dates near the pincode.
> 3. Still not getting mails.
> 
> <img alt="Slots" width="1420" src="https://user-images.githubusercontent.com/16135238/116850440-4c9fe900-ac0e-11eb-922e-86cc884098bd.png">
> 
> Any help

**Comment by Pratik2125** [comment]
@sid8840 yes, it has slots available

**Comment by sid8840** [comment]
> @sid8840 yes, it has slots available

Use this PR
https://github.com/jithin-zachariah/VaccineAvailabilityNotifier

**Comment by mrsus** [comment]
Please run `pm2 logs vaccineNotifier` to see any error occurred in script.
